modules/rosa-sts-terraform-considerations.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * rosa_planning/rosa-understanding-terraform.adoc
+:_mod-docs-content-type: CONCEPT
+[id="rosa-sts-terraform-considerations_{context}"]
+= Considerations when using Terraform
+
+In general, using Terraform to manage cloud resources should be done with the expectation that any changes should be done using the Terraform methodology. Use caution when using resources outside of Terraform, such as the AWS console or Red Hat console, to modify resources created by Terraform. Using tools outside Terraform to manage resources that are already managed by Terraform introduces configuration drift from your declared Terraform configuration.
```

Comment:
We need to use the word 'resources' to describe cloud resources, so let's not use it to describe the consoles, which could be called 'tools'.